test/testbed/scenarios/fixtures/command_api.go

```diff
@@ -61,15 +61,15 @@ integrations:
       apis:
         - event_type: LinuxFileList
           commands:
-            - run: ls -l /etc/apt/
+            - run: ls -l --time-style="+%Y" /etc/apt/
```

Comment:
E2E tests where failing because the displayed data, ls date format varies depending on the local time: https://unix.stackexchange.com/questions/237531/why-does-ls-all-show-time-for-some-files-but-only-year-for-others